miageWebApp/src/app/cours/cours.component.ts

```diff
@@ -0,0 +1,75 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup } from '@angular/forms';
+
+import { Cours } from '../model/cours';
+import { CoursService } from '../services/cours.service';
+import { ThrowStmt } from '@angular/compiler';
+
+@Component({
+  selector: 'app-cours',
+  templateUrl: './cours.component.html',
+  styleUrls: ['./cours.component.scss']
+})
+
+export class CoursComponent implements OnInit {
+  myClasses: Cours[] = [];
+  addClassForm: FormGroup;
```

Comment:
```suggestion
  classForm: FormGroup;
```

Reply:
yes but what if after you will use this form to display the information in an other bloc, it's just a class form, not specially an "add" form

Reply:
corrected